tests/encoding_case.phpt

```diff
@@ -8,6 +8,5 @@ require_once('Mail/mimePart.php');
 print Mail_mimePart::quotedPrintableEncode($test, 7);
 ?>
 --EXPECT--
-Here's=
-=09
+Here's=09
```

Comment:
@ixs Email is not really my problem domain anymore. I see you adjusted another test, is that save to do?

@pear/mail-components Anyone else wants to weigh in? LGTM otherwise.

Reply:
@till 😁 That test broke after my change.
I had a hard and long look at it, fixed a bug in my code and then looked again at the test.

I was just explaining what exactly the old code did and why it did it wrong here... But while writing that down, I'm having a few doubts about and think I might have missed an interesting case... So let me have another hard look at this test and get back to you in a few. 😝